katalog/etcd-backup-s3/README.md

```diff
@@ -0,0 +1,122 @@
+# ETCD Backup S3
+
+This package provides an automated solution for backing up ETCD data to an S3-compatible storage service using a Kubernetes CronJob.
+
+## Overview
+
+This package creates a CronJob that:
+
+1. Takes a snapshot of the ETCD database
+2. Uploads the snapshot to an S3-compatible storage target
+3. Cleans up old backups based on a configurable retention policy
+
+The job is scheduled to run on control plane nodes and uses `rclone` to manage the S3 uploads and lifecycle management.
```

Comment:
nodes in plural? Does it run on several nodes at the same time?

Reply:
Nodes in plural because there's no guarantee that it will run on the same control plane node each time, but there's no concurrent execution.

Reply:
Maybe let's change it as one of the available control plane nodes